core/src/main/java/hudson/WebAppMain.java

```diff
@@ -90,6 +89,11 @@
             }
         }
     };
+
+    //written for the requirements of JENKINS-50669
+    public int getDefaultRingBufferSize() {
```

Comment:
this method is returning the current size and should be static for evergreen-plugin be able to call it. Please, remove this method and we can call directly the static field in the `RingBufferLogHandler`